edbdeploy/data/templates/config.yml.j2

```diff
@@ -143,9 +141,7 @@ instances:
   public_ip:  {{ pooler_server['public_ip'] }}
   private_ip: {{ pooler_server['private_ip'] }}
   role:
-  - pgbouncer
   - harp-proxy
-  - etcd
```

Comment:
I think we should keep the `etcd` role because we rely on `etcd` as consensus protocol (`harp_consensus_protocol: etcd`)

Reply:
It is interesting. I think CentOS comes with `etcd`; therefore, it didn't affect my testing. However, HARP for sure uses the etcd. Also, the new `tpaexec`'s config.yml doesn't include etcd.
```
- Name: upwards
  location: b
  node: 12
  role:
  - harp-proxy
  volumes:
  - device_name: /dev/xvdf
    volume_type: none
- Name: ushers
  location: b
  node: 13
  role:
  - harp-proxy
  volumes:
  - device_name: /dev/xvdf
    volume_type: none
```
I have made these changes as per the latest version of `tpaexec,` and no issues were shown in my testing.

Reply:
Where are located the `etcd` instances in this case? We need at least 3 `etcd` instances and we only have 2 pooler nodes.

Reply:
`etcd`s are located on BDR nodes.
```
[root@epas1 ~]# rpm -qa|grep etcd
[root@epas1 ~]# ps -eaf|grep etcd
root       31596       1  1 Dec23 ?        00:11:30 /usr/bin/etcd
```
```
cluster:
  name: bdrpot

dcs:
  driver: "etcd"
  request_timeout: 250
  ssl: "off"
  ssl_ca_file: ""
  ssl_cert_file: ""
  ssl_key_file: ""
  endpoints:
  - epas1:2379
  - epas2:2379

manager:
  name: "epas1"
  log_level: debug```